assets/js/components/KeyMetrics/ChipTabGroup/Chip.js

```diff
@@ -36,15 +40,33 @@ export default function Chip( {
 	onClick,
 	selectedCount = 0,
 } ) {
+	function Icon() {
+		return null;
+	}
+
+	switch ( slug ) {
+		case KEY_METRICS_CURRENT_SELECTION_GROUP_SLUG:
+			Icon = function ( args ) {
+				return <CheckMark { ...args } />;
+			};
+			break;
+		case KEY_METRICS_SUGGESTED_GROUP_SLUG:
+			Icon = function ( args ) {
+				return <StarFill { ...args } />;
+			};
+			break;
+	}
```

Comment:
Not sure why we need to wrap icons into functions here. Ideally, we can simply use a map to define which icon we should use. Something like this:

```js
const icons = {
    KEY_METRICS_CURRENT_SELECTION_GROUP_SLUG: CheckMark,
    KEY_METRICS_SUGGESTED_GROUP_SLUG: StarFill,
};

export default function Chip( { ... } ) {
    const Icon = icons[ slug ] || Null;

    ...
}
```

Reply:
This has been addressed in recent commits, thanks @eugene-manuilov